IntelliTect.TestTools.Data/InMemoryLogger.cs

```diff
@@ -43,7 +43,7 @@ public bool IsEnabled(LogLevel logLevel)
         /// </summary>
         public IDisposable BeginScope<TState>(TState state)
         {
-            throw new NotImplementedException();
+            return null;
```

Comment:
I think you want `return this;` don't you? Otherwise scoped invocations I would expect to fail.

Reply:
It looks like null return is expected, looking at other implementations. https://github.com/horsdal/InMemoryLogger/blob/master/src/InMemoryLogger.cs